benchmarks/testminer.py

```diff
@@ -106,6 +108,8 @@ def get_test_job_details(self, job_id):
         if content['multinode_definition']:
             definition = json.loads(content['multinode_definition'])
         details.update({"definition": str(json.dumps(definition))})
+        details['metadata'] = extract_metadata(definition)
+        details['name'] = extract_name(definition)
         for action in definition['actions']:
             if action['command'].startswith("submit_results"):
                 if 'stream' in action['parameters'].keys():
```

Comment:
Github doesn't allow me to comment lines below. There is "if 'metadata'..." line (line 117) which is probably not needed as metadata is extracted in line 111.